### ISSUE TITLE
[Bug]:  RuntimeError: mm_sd_v15.ckpt hash mismatch. You probably need to re-download the motion module.

### ISSUE THREAD

**Issue by D3voz** [continue-revolution/sd-webui-animatediff#37]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### Have you read FAQ on README?

- [X] I have updated WebUI and this extension to the latest version

### What happened?

*** Error running before_process: C:\github\diffusers\stable-diffusion-webui\extensions\sd-webui-animatediff\scripts\animatediff.py
    Traceback (most recent call last):
      File "C:\github\diffusers\stable-diffusion-webui\modules\scripts.py", line 487, in before_process
        script.before_process(p, *script_args)
      File "C:\github\diffusers\stable-diffusion-webui\extensions\sd-webui-animatediff\scripts\animatediff.py", line 138, in before_process
        self.inject_motion_modules(p, model)
      File "C:\github\diffusers\stable-diffusion-webui\extensions\sd-webui-animatediff\scripts\animatediff.py", line 87, in inject_motion_modules
        raise RuntimeError(f"{model_name} hash mismatch. You probably need to re-download the motion module.")
    RuntimeError: mm_sd_v15.ckpt hash mismatch. You probably need to re-download the motion module.

### Steps to reproduce the problem

1. installed animatediff extension
2. download the motion model 
3. enabled and typed the prompt 

### What should have happened?

no error i guess?

### Commit where the problem happens

webui: 
extension: 

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
no
```

### Console logs

```Shell
2023-07-27 05:38:29,172 - AnimateDiff - INFO - AnimateDiff process start with video Max frames 16, FPS 8, duration 2.0,  motion module mm_sd_v15.ckpt.
*** Error running before_process: C:\github\diffusers\stable-diffusion-webui\extensions\sd-webui-animatediff\scripts\animatediff.py
    Traceback (most recent call last):
      File "C:\github\diffusers\stable-diffusion-webui\modules\scripts.py", line 487, in before_process
        script.before_process(p, *script_args)
      File "C:\github\diffusers\stable-diffusion-webui\extensions\sd-webui-animatediff\scripts\animatediff.py", line 138, in before_process
        self.inject_motion_modules(p, model)
      File "C:\github\diffusers\stable-diffusion-webui\extensions\sd-webui-animatediff\scripts\animatediff.py", line 87, in inject_motion_modules
        raise RuntimeError(f"{model_name} hash mismatch. You probably need to re-download the motion module.")
    RuntimeError: mm_sd_v15.ckpt hash mismatch. You probably need to re-download the motion module.
```

**Comment by continue-revolution** [comment]
Your motion module might only be partially downloaded. You can disable hash check in Settings/ AnimateDiff to see if it is my problem. If you can use it normally, I will fix it when I have time.

**Comment by continue-revolution** [comment]
Alright, it is my problem. Pls temporarily uncheck the hash check in settings and I will fix after several hours.

**Comment by continue-revolution** [comment]
fixed in [bcca007a1ba700f1cd231698c00e460a63581171](https://github.com/continue-revolution/sd-webui-animatediff/commit/bcca007a1ba700f1cd231698c00e460a63581171)
